sllm/serve/model_downloader.py

```diff
@@ -42,6 +43,9 @@ def download_transformers_model(
 ) -> bool:
     storage_path = os.getenv("STORAGE_PATH", "./models")
     model_path = os.path.join(storage_path, "transformers", model_name)
+    tokenizer_path = os.path.join(
+        storage_path, "transformers", model_name + "_tokenizer"
```

Comment:
I think it's fine to save the tokenizer with the model checkpoints, i.e., in the same directory.

Reply:
How about we save the tokenizer in "storage_path/transformers/model_name/tokenizer"?

Reply:
Thus, when we read the tokenizer from the local file, we can check if there's such a local dir so we can fall back to default if not. (to not break the previous saved model path)

Reply:
Sounds good